### DISCUSSION TITLE
How do I make functions in a loop execute only once per second?

### DISCUSSION THREAD

**Discussion by wxwow90** [pixijs/pixijs#7699] [Q&A] [answered]
How do I make functions in a loop execute only once per second?
Using setInterval or setTimeout will refresh repeatedly.

**Comment by miltoncandelero** [comment, marked as answer]
accumulate your delta time until it becomes greater than 1000. When it does, reset it to 0, execute your thingy and then start accumulating again

https://www.pixiplayground.com/#/edit/zhhD3CWRIshj8c5xiLUJi

**Reply by wxwow90** [reply]
thank you very much.
